packages/cli/src/commands/build.ts

```diff
@@ -265,7 +268,7 @@ export default {
             (key.name == "c" && key.ctrl)
           ) {
             await watcher.stop();
-            await dockerLock.release();
+            // await dockerLock.release();
```

Comment:
This is needed. If the process is `kill()`ed, then the file lock will still exist, causing subsequent calls to halt indefinitely.

Reply:
That's just debugging leftover

Reply:
Thanks for catching it